src/Raven.Server/Documents/TcpHandlers/SubscriptionConnection.cs

```diff
@@ -1021,6 +1135,9 @@ public void Dispose()
                 Stats.Dispose();
 
                 RecentSubscriptionStatuses?.Clear();
+                
+                _actualConnectionScope.Dispose();
```

Comment:
please check if disposing parent will dispose children as well then we can remove field _actualConnectionScope
from this class and have reference to parent scope only.

Reply:
`_actualConnectionScope ` _is_ the child scope..  (currently renamed to  `_activeConnectionScope`)
 and we also have below ` _parentConnectionScope.Dispose();`